Gemfile

```diff
@@ -4,3 +4,12 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'minitest', '~> 5.5'
+gem 'mocha', '~> 0.14'
+gem 'rake', '>= 12.3.3'
+gem 'shoulda', '~> 2.11'
+gem 'simplecov', '< 0.22.0'
+gem 'simplecov-lcov', '> 0.7.0'
+gem 'systemu', '~> 2'
+gem 'timecop', '~> 0.9'
```

Comment:
It is a best practice now (according to Rubocop) to move development dependencies out of the `gemspec` and into the `Gemfile`. Note the `Gemfile` is only for development not loaded when users include the gem in a project.